services/country.js

```diff
@@ -1,17 +1,19 @@
 import { all, spread } from 'axios';
 
-import { cartoRequest } from 'utils/request';
+import { cartoRequest, dataRequest } from 'utils/request';
 import { getGadm36Id } from 'utils/gadm';
 
+const GADM_DATASET = '/dataset/gadm_administrative_boundaries/v4.1/query';
```

Comment:
@manukala6 @gtempus @dmannarino this service is responsible for populating the regions list.